### PR TITLE
Add stats to success screen

### PR DESCRIPTION
**Pull request by psugihara**
part of #102

### CODE REVIEW

File: src/anybody.js [hunk 1]
```diff
@@ -594,7 +630,44 @@ export class Anybody extends EventEmitter {
...
+    ]
+
+    const SPEED_BOOST = [
+      1, // <10s left
+      1, // <20s left
+      1, // <30s left
+      1, // <40s left
+      2, // <50s left
+      4 // < 60s left
+    ]
```
Comment (okwme) [inline src/anybody.js:657]: Just updated contract with these values (#110)

```suggestion
    const SPEED_BOOST = [
      6, // <10s left
      5, // <20s left
      4, // <30s left
      3, // <40s left
      2, // <50s left
      1 // < 60s left
    ]
```

**Comment by okwme** [comment]
hell yea
Screenshot 2024-04-11 at 13 34 44

this isn't R4R tho, or?  
looks like your testing duration 2 seconds got left in

File: src/visuals.js [hunk 2]
```diff
@@ -1226,6 +1269,7 @@ export const Visuals = {
       body.witherSteps++
       if (body.witherSteps > WITHERING_STEPS) {
         this.witheringBodies = this.witheringBodies.filter((b) => b !== body)
+        p.pop()
```
Comment (okwme) [inline src/visuals.js:1272]: why is this here?

File: src/anybody.js [hunk 3]
```diff
@@ -238,16 +242,48 @@ export class Anybody extends EventEmitter {
...
   handleGameOver = ({ won }) => {
+    if (this.handledGameOver) return
+    this.handledGameOver = true
+
     this.witherAllBodies()
     this.sound?.playGameOver({ won })
     this.gameOver = true
-    this.won = won
-    this.setShowPlayAgain()
+    this.won = true
```
Comment (okwme) [inline src/anybody.js:251]: don't forget to remove this when ready to merge

**Comment by okwme** [comment]
@evvvritt 's feedback on calculations:

Screenshot 2024-04-11 at 14 01 27

**Comment by okwme** [comment]
also maybe speed bonus as `-` instead of `x1`

File: src/visuals.js (hunk 2 above)
Comment (psugihara) [reply src/visuals.js:1273]: this is a bug I found that can interfere with game over text and other stuff. `p.push` then translate is called at the top of the loop and the next line here is `continue`, so it misses the `p.pop` at the end of the loop body and ends up keeping the translation for subsequent drawing.

**Comment by psugihara** [comment]
think this is good to merge now @okwme. I'll work on adding buttons for claim and play again + FOMO text next.

**Comment by psugihara** [comment]
Screenshot 2024-04-11 at 8 58 20 AM

it might read a little clearer if the numbers were right aligned
